### ISSUE TITLE
AssertionError: Number must be an unsigned integer, not a float 

### ISSUE THREAD

**Issue by brentru** [adafruit/Adafruit_CircuitPython_RSA#11]
as discussed in https://github.com/adafruit/Adafruit_CircuitPython_RSA/pull/10, the simpletest for this library fails:

on 5.0
```
Auto-reload is on. Simply save files over USB to run them or enter REPL to disable.
code.py output:
Generating keypair...
Encrypting message...
Traceback (most recent call last):
  File "code.py", line 16, in <module>
  File "/lib/adafruit_rsa/pkcs1.py", line 173, in encrypt
  File "/lib/adafruit_rsa/transform.py", line 181, in int2bytes
AssertionError: Number must be an unsigned integer, not a float.
```

and on 4.1.2
```
Auto-reload is on. Simply save files over USB to run them or enter REPL to disable.
code.py output:
Generating keypair...
Encrypting message...
Traceback (most recent call last):
  File "code.py", line 16, in <module>
  File "/lib/adafruit_rsa/pkcs1.py", line 173, in encrypt
  File "/lib/adafruit_rsa/transform.py", line 177, in int2bytes
AssertionError: Number must be an unsigned integer, not a float.```
```

**Comment by evaherrada** [comment]
I'll take a look at this today and see if it's something I can fix

**Comment by evaherrada** [comment]
Fixed by #12
